pennylane_lightning/src/tests/TestKernels.hpp

```diff
@@ -3,11 +3,26 @@
  * @brief We define test kernels. Note that kernels not registered to
  * AvailableKernels can be also tested by adding it to here.
  */
-#include "GateImplementationsLM.hpp"
-#include "GateImplementationsPI.hpp"
+#include "cpu_kernels/GateImplementationsLM.hpp"
+#include "cpu_kernels/GateImplementationsPI.hpp"
```

Comment:
Should we use a nested CMakeLists file, with an associated interface target? This may allow us to avoid explicitly relative paths which can be a challenge when building extensions with libraries.